build/Rulesets/Roslyn_BuildRules.ruleset

```diff
@@ -102,7 +102,7 @@
     <Rule Id="CA1308" Action="None" />
     <Rule Id="CA1810" Action="None" />
     <Rule Id="CA1816" Action="None" />
-    <Rule Id="CA1825" Action="Warning" />
+    <Rule Id="CA1825" Action="None" />
```

Comment:
Which one was this?

Reply:
Avoid empty allocations - uses an IOperation API that was removed, should be fixed when we move the analyzer to use new APIs.